src/reducers/activePaletteReducer.js

```diff
@@ -0,0 +1,8 @@
+export const activePaletteReducer = (state = [], action) => {
+  switch(action.type) {
+    case 'SET_ACTIVE_PALETTE':
+      return action.palette
+    default:
+      return state
+  }
+}
```

Comment:
I'm thinking that we could also use the reducer potentially to sort through what is locked and what is not...?